source/docs/javascript_usage.md

```diff
@@ -4,6 +4,26 @@ title: Usage in Javascript
 Embark includes in your dapp the EmbarkJS library. This library abstracts different functionality so you can easily & quickly build powerful dapps that leverage different decentralized technologies.
 Embark will automatically initialize EmbarkJS with the configurations set for your particular environment.
 
+### Provider
+
+According to [EIP1102](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1102.md), Dapps MUST request account access and Embark allows you to choose how you want to deal with request access.
+
+In the smart contracts configuration you can find this snippets:
+<pre><code class="javascript">
+...
+  // Automatically call `ethereum.enable` if true.
+  // If false, the following code must run before sending any transaction: `await EmbarkJS.enableEthereum();`
```

Comment:
Answer: Yes if you use await it will block until the user accepts.
When user denied you will need a catch.
I am adding it to the doc with example

Reply:
@bgits Since it's using `await` it's a promise based API. `enableEthereum()` delegates to `web3.enable()` which is deprecated in favour of `web3.eth_requestAccounts()`: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1102.md#eth_requestAccounts

The API documentation is a little counterintuitive here. The signature says it returns `Promise<string>` but the docs say it returns either `Array<Account>` or `Error` in case of rejection. I can imagine that the error case is just an error message (therefore `Promise<String>`).